addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -1214,16 +1223,32 @@ options.registry.WebsiteFieldEditor = FieldEditor.extend({
                 <we-button data-select-data-attribute="!fileSet">${_t("Is not set")}</we-button>
             </we-select>
         `)[0]);
+        const recursiveFindCircular = (el) => {
+            const dependencyEl = this._getDependencyEl(el);
+            if (!dependencyEl) {
+                return false;
+            }
+            if (dependencyEl.closest('.s_website_form_field').dataset.visibilityDependency === this._getFieldName()) {
```

Comment:
`dependencyEl.closest('.s_website_form_field')` should be put inside a temp variable :+1:

Reply:
Done, indeed